sidebars.js

```diff
@@ -164,6 +164,27 @@ module.exports = {
           type: 'category',
           label: 'Develop on Calamari',
           items: [
+            {
+              type: 'category',
+              label: 'How to Mint zkSBT',
+              items: [
+                {
+                  type: 'doc',
+                  label: 'How to Mint zkSBT',
```

Comment:
I think this menu should after Wallet, in third place. Because it depends on wallet.